ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -366,7 +366,10 @@ pub(crate) enum Instruction {
         args: Vec<Operand>,
     },
     #[deku(id = "5")]
-    Br { succ: BBlockIdx },
+    Br {
+        /// The block this branch points to.
```

Comment:
I'm a bit confused: `succ` normally means "the block after the current block in the layout" but the comment suggests it's the target branch?

Reply:
Well, LLVM calls them successor blocks. They are the successor blocks of the current block in terms of control flow, not in terms of their index in the block vector.

Reply:
Ouch, not great terminology, but at least I now know!

Reply:
I can rename it to `target_bb` if you like. I'm not fussed.

Reply:
Using LLVM's terminology here does make sense, so let's keep it.